polyaxon/templates/api-deployment.yaml

```diff
@@ -113,7 +119,7 @@ spec:
           httpGet:
             path: /healthz/
             port: {{ .Values.streams.service.internalPort }}
-            scheme: {{ .Values.api.service.scheme | default "HTTP" }}
+            scheme: {{ .Values.streams.service.scheme | default "HTTP" }}
```

Comment:
You can revert this change, the streams service only defines a port, the scheme for all APIs endpoints are set on the api service definition.

Same for the other change in [line 132](https://github.com/polyaxon/polyaxon-chart/pull/51/files#diff-92d5c80b2b2de44841d0bc82b99c7b3cR132)

Reply:
Readiness/liveness probes for streams service don't work when SSL is enabled and the probes for API service are configured to use HTTPS scheme.

Reply:
I see, it's weird, never had that issue. But it's ok.

Reply:
It's not ok because the chart was deployed with FAILED status.

Does streams service use https when SSL is enabled?